ahorcado.py

```diff
@@ -0,0 +1,262 @@
+import random
+import os
+import grafico
+#################    FUNCIONES DEL MENU   ####################
+def menu_option_2(battery_file):
+    with open(battery_file, "r") as f:
+        new_file = [] 
+        for line in f:
+            new_file.append(line.strip('\n'))   ### LEVANTA LAS PALABRAS DEL TXT
+
+    while True:               ### Voy a suponer q el usuario ingresa la palabra de la manera adecuada
+        new_word = input('Ingresa una palabra sin espacios ni caracteres raros\n')
+        if new_word == '1':   ### Sino tendria que hacer una funcion que compare todos los
+            break             ### caracteres a ver si son validos
+        elif not new_word in new_file:
+            new_file.append(new_word)
+            print('Agregado!, Ingresa otra palabra o \"1\" para salir')
+        else:
+            print('Esa palabra ya está en la lista')
+
+
+def menu_option_3(file):
+    with open(file, "r") as f:
+        for line in f:
+            print(line.strip("\n"))
+
+###############################################################
+def no_se_que_func_tiene_python_para_esto(y):
+    result = ""
+    for x in y:
+        result += x.upper() + " "
+    return result
+
+###############################################################
+def busca_indices(letra, quest, muestra):
+    muestra = list(muestra)
+    for index, x in enumerate(quest):  # genera en index el indice, y en x el objeto dentro de quest
+        if x == letra:
+            muestra[index] = letra
+    return muestra
+################################################################
+
+def letras_utilizadas(letra, lista):
+    if letra in lista:
+        return True
```

Comment:
esta funcion esta demas... podrias hacer directamente `if letra in lista` en vez de `if letras_utilizadas(letra, lista)`

Reply:
si, antes lo tenia así y vos me dijiste que lo pase a funcion xD

Reply:
mmmm tal vez hablando te dije de hacer una funcion para chequear que la letra este en la lista, pero no era algo explicitamente de "movelo a una funcion si es una linea"... En este caso esto se simplifica a 1 linea:

```
def letras_utilizadas(letra, lista):
    return letra in lista
```

Aca si no tiene sentido hacer una funcion porque Python ya te lo da facilmente con el `in`.

Hay otros casos donde si queres hacer una funcion de 1 sola linea, y es cuando es algo mas complejo y lo usas en varios lados, entonces queres tenerlo en una funcion y cambiar solo la funcion en caso de ser necesario, en vez de cambiar esa linea en todo el codigo. Pero este no es el caso

Reply:
solved